frontend/views/containers/user-settings/NotificationVolume.vue

```diff
@@ -0,0 +1,116 @@
+<template lang='pug'>
+.c-notification-volume
+  i18n.c-title Volume
+  .c-volume-slider-container
+    slider-continuous.c-volume-slider(
+      uid='notification-volume'
+      :min='config.sliderMin'
+      :max='config.sliderMax'
+      :unit='config.sliderUnit'
+      :hideText='true'
+      :value='ephemeral.volume'
+      @input='handleVolumeUpdate'
+    )
+
+    .c-volume-value {{ ephemeral.volume }}{{ config.sliderUnit }}
+
+  audio(ref='exampleAudio'
+    src='/assets/audio/msg-received.mp3'
+    type='audio/mpeg'
+  )
+</template>
+
+<script>
+import { mapMutations } from 'vuex'
+import SliderContinuous from '@components/SliderContinuous.vue'
+import { debounce } from 'turtledash'
+
+export default ({
+  name: 'NotificationVolume',
+  components: {
+    SliderContinuous
+  },
+  data () {
+    return {
+      config: {
+        sliderMin: 0,
+        sliderMax: 100,
+        sliderUnit: '%'
+      },
+      ephemeral: {
+        volume: 100
+      }
+    }
+  },
+  computed: {
+    volumeFromStore () {
+      return this.$store.getters.notificationVolume || 1
+    }
```

Comment:
`o3-mini-high` caught an issue here: since `notificationVolume` can be `0` now, that would be false, and this would then return `1` - not what we intended.

So the suggestion would be:

```js
// check done this way because `0` is falsy
return this.$store.getters.notificationVolume != null ? this.$store.getters.notificationVolume : 1
```

Note that the same change must be done in `Background.vue` as well, as that also has this same computed property.

Reply:
which can be simplified to
```js
return this.$store.getters.notificationVolume ?? 1
```